index.d.ts

```diff
@@ -1,3 +1,31 @@
+export interface TimeoutOptions {
```

Comment:
@sindresorhus I kinda wish p-timeout accepted this object instead of 4 parameters, like

`pTimeout(promise, options)`

So this type would live in that package too

Reply:
I agree. That would be nicer.

@jopemachine Interested in performing that change?

Reply:
> Interested in performing that change?

Sure, I will try to create a related PR to `p-timeout` first.

Reply:
> Sure, I will try to create a related PR to p-timeout first.

Bump :)

Reply:
https://github.com/sindresorhus/p-timeout/releases/tag/v6.0.0